### ISSUE TITLE
[Request] Buddysma!! ~Smartphone Start! Buddyfight

### ISSUE THREAD

**Issue by kdj0692** [vgmstream/vgmstream#1491]
**READ THIS AND THEN DELETE IT. Do not submit issues related to converting audio _TO_ video game music formats. This goes beyond the scope of this project.**

https://drive.google.com/file/d/111YMmYssSzAt_-a0Z0-25QbZVMjaOyBJ/view?usp=drivesdk

**Comment by Bonboon229** [comment]
Someone close this issue, please.
Has a link to a site know for viruses.

**Comment by kdj0692** [comment]
> Someone close this issue, please. Has a link to a site know for viruses.

This guy is lying. I installed the app myself, no virus found.

**Comment by Bonboon229** [comment]
I repeat, please close the issue.

**Comment by kdj0692** [comment]
> I repeat, please close the issue.

Are you making fun of me?  It's fun. LOL.

**Comment by Bonboon229** [comment]
I'm not.
First point. Don't link to APK's here.
Second Point. The site is a virus riddled mess.

**Comment by kdj0692** [comment]
https://download.apkcombo.com/com.bushiroad.buddyfight/%E3%83%90%E3%83%87%E3%82%A3%E3%82%B9%E3%83%9E%20~%E3%82%B9%E3%83%9E%E3%83%9B%E3%81%A7%E3%82%B9%E3%82%BF%E3%83%BC%E3%83%88%EF%BC%81%E3%83%90%E3%83%87%E3%82%A3%E3%83%95%E3%82%A1%E3%82%A4%E3%83%88%EF%BC%81~%20%E3%83%88%E3%83%AC%E3%83%BC%E3%83%87%E3%82%A3%E3%83%B3%E3%82%B0%E3%82%AB%E3%83%BC%E3%83%89%E3%82%B2%E3%83%BC%E3%83%A0_2.0.0_apkcombo.com.apk?ecp=Y29tLmJ1c2hpcm9hZC5idWRkeWZpZ2h0LzIuMC4wLzQwLmFmMjg2NjRkNjFlMjdjOGFiYWUzZjBjOTFiMGIyMTVlNjM4OTc2ZDUuYXBr&iat=1708273335&sig=337f452e8332d8d8e7caca9cfa672575&size=77188948&from=cf&version=latest&lang=en&fp=7dfe6acc80c1e4d3dcba609e2b69583c&ip=58.126.194.159

Please install it yourself and evaluate it.

**Comment by kdj0692** [comment]
https://download.apkcombo.com/com.bushiroad.buddyfight/%E3%83%90%E3%83%87%E3%82%A3%E3%82%B9%E3%83%9E%20~%E3%82%B9%E3%83%9E%E3%83%9B%E3%81%A7%E3%82%B9%E3%82%BF%E3%83%BC%E3%83%88%EF%BC%81%E3%83%90%E3%83%87%E3%82%A3%E3%83%95%E3%82%A1%E3%82%A4%E3%83%88%EF%BC%81~%20%E3%83%88%E3%83%AC%E3%83%BC%E3%83%87%E3%82%A3%E3%83%B3%E3%82%B0%E3%82%AB%E3%83%BC%E3%83%89%E3%82%B2%E3%83%BC%E3%83%A0_2.0.0_apkcombo.com.apk?ecp=Y29tLmJ1c2hpcm9hZC5idWRkeWZpZ2h0LzIuMC4wLzQwLmFmMjg2NjRkNjFlMjdjOGFiYWUzZjBjOTFiMGIyMTVlNjM4OTc2ZDUuYXBr&iat=1708273335&sig=337f452e8332d8d8e7caca9cfa672575&size=77188948&from=cf&version=latest&lang=en&fp=7dfe6acc80c1e4d3dcba609e2b69583c&ip=

Please install it yourself and evaluate it.

**Comment by Bonboon229** [comment]
I'm not talking about the app you idiot. I'm talking about the site.

**Comment by kdj0692** [comment]
> I'm not talking about the app you idiot. I'm talking about the site.

I uploaded the app file to Google Drive and replaced the link. 
Rest assured that it is a virus-free app, and please do not request to delete this article anymore.

**Comment by bnnm** [comment]
- Make clear what your issue is with an actual explanation
- Do no post random links without an explanation of what they are
- Read and understnad the text that appears when making an issue, that you copied, and comply with it

**Comment by applecuckoo** [comment]
Hi! I unzipped the contents of the file looking around for any sounds, and I believe the purpose of @kdj0692 opening this issue is for an hca key request. I've already gone and collected the sound files and the libraries, which I've stored in [Google Drive](https://drive.google.com/file/d/1DTmba7FRFy503Z17n10BLBHz68IV6-5R/view?usp=drive_link).

**Comment by kdj0692** [comment]
> Hi! I unzipped the contents of the file looking around for any sounds, and I believe the purpose of @kdj0692 opening this issue is for an hca key request. I've already gone and collected the sound files and the libraries, which I've stored in [Google Drive](https://drive.google.com/file/d/1DTmba7FRFy503Z17n10BLBHz68IV6-5R/view?usp=drive_link).

Thank you very much for your hard work.

**Comment by kdj0692** [comment]
> Hi! I unzipped the contents of the file looking around for any sounds, and I believe the purpose of @kdj0692 opening this issue is for an hca key request. I've already gone and collected the sound files and the libraries, which I've stored in [Google Drive](https://drive.google.com/file/d/1DTmba7FRFy503Z17n10BLBHz68IV6-5R/view?usp=drive_link).

That's right, I made the request because I needed a key. I

**Comment by bnnm** [comment]
> Hi! I unzipped the contents of the file looking around for any sounds, and I believe the purpose of @kdj0692 opening this issue is for an hca key request. I've already gone and collected the sound files and the libraries, which I've stored in [Google Drive](https://drive.google.com/file/d/1DTmba7FRFy503Z17n10BLBHz68IV6-5R/view?usp=drive_link).

Thanks, though that is fairly evident (we can figure out the weirdest formats and codecs, so this isn't exactly taxing).

That I was trying to imply is this: when somebody makes an issue in this repo they should write some info and put effort into comunicating, and essentially request help from strangers. Sometimes the person making the issue doesn't speak English well, that is fine if as long as the effort is there. Instead here I get just a name and a link with the implication "figure it out", like if this repo was handled by AI bots. User did not even read the "remove this" text.

In other words, I could use my precious little free time figuring this out or getting this hca key for somebody that doesn't even treat me like a human being, or I could just go play videogames or something.
